temporalio/lib/temporalio/testing/workflow_environment.rb

```diff
@@ -322,7 +322,7 @@ def auto_time_skipping_disabled(&)
           raise 'Block required' unless block_given?
           return super unless supports_time_skipping?
 
-          already_disabled = @auto_time_skipping
+          already_disabled = !@auto_time_skipping
```

Comment:
Thanks! I think our logic at https://github.com/temporalio/sdk-dotnet/blob/d48377d698b2a2cb7a8c58d181959df473875314/src/Temporalio/Testing/WorkflowEnvironment.cs#L323-L338 is wrong (and https://github.com/temporalio/sdk-python/blob/e3603989f081bb812d883189be5f5ac9aa127089/temporalio/testing/_workflow.py#L464-L472 is right), so time to fix .NET too heh.